wire/wire.go

```diff
@@ -13,22 +13,60 @@
 // limitations under the License.
 
 // Package wire contains directives for Wire code generation.
+// For an overview of working with Wire, see the user guide at
+// https://github.com/google/go-cloud/blob/master/wire/README.md
+//
+// The directives in this package are used as input to the Wire code
+// generation tool. The entry point of Wire's analysis are injector functions:
+// function templates denoted by only containing a call to Build. Build names
```

Comment:
How about instead of "Build names", "The arguments to Build define a set of providers" or "are a set of providers"?

Reply:
Done.